src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -230,9 +240,10 @@ export const getThorchainSaversWithdrawQuote = async ({
   return quoteData
 }
 
-export const getMidgardPools = async (): Promise<MidgardPoolResponse[]> => {
+export const getMidgardPools = async (period?: string): Promise<MidgardPoolResponse[]> => {
```

Comment:
`7d` needed since stablecoin savers are only live for a few days - we would get 0% APY otherwise

Reply:
could be cool to type period as a sting union so we know what valid values there are, e.g 
```
'1d' | '7d' | '30d'
```

Reply:
cd70b34e30